tests20/python_client/base/client_request.py

```diff
@@ -116,29 +128,46 @@ def get_index_param(self, request):
         yield request.param
 
     def _connect(self):
-        """ Testing func """
-        self.connection.add_connection(default={"host": param_info.param_host, "port": param_info.param_port})
-        res = self.connection.connect(alias='default')
+        """ Add an connection and create the connect """
+        self.connection_wrap.add_connection(default={"host": param_info.param_host, "port": param_info.param_port})
+        res = self.connection_wrap.connect(alias='default')
         return res
 
-    def _collection(self, name=None, data=None, schema=None, check_res=None, **kwargs):
-        """ Testing func """
-        self._connect()
+    def _collection(self, name=None, data=None, schema=None, check_res=None, c_object=None, **kwargs):
+        """ Init a collection and return the object of collection """
         name = cf.gen_unique_str("ApiReq") if name is None else name
         schema = cf.gen_default_collection_schema() if schema is None else schema
-        collection, _ = self.collection.collection_init(name=name, data=data,
-                                                        schema=schema, check_res=check_res, **kwargs)
-        assert name == collection.name
-        return collection
-
-    def _partition(self, name=None, descriptions=None, **kwargs):
-        """ inti a partition in a collection """
-        # self._connect()
-        m_collection = self._collection()
+        c_object = self.collection_wrap if c_object is None else c_object
```

Comment:
I don't think this is useful. _collection is designed for quickly initialize a collection with everything default.